src/pages/session/components/AppBar.tsx

```diff
@@ -14,11 +15,9 @@ const AppBar = () => {
   const dispatch = useAppDispatch();
   const [anchorEl, setAnchorEl] = useState<HTMLElement|null>(null);
 
-  const { profile } = useAppSelector((state) => state.auth);
-
   const getProfileAddress = () => {
-    if (!profile || !profile.address) return '...';
-    return `${profile.address.slice(0,4)}...${profile.address.slice(-4)}`;
+    const address = localStorage.getItem(LocalStorageKeys.address);
```

Comment:
@EehMauro , 
We need to make use of useLocalStorage hook for modularisation setting and getting values from localStorage